### ISSUE TITLE
Lost wluna on shuttle

### ISSUE THREAD

**Issue by danielgolf272** [terra-money/bridge-web-app#213]
Hi there,
I tried to bridge wluna to terra on 6/6/22. I had been talking to Coinbase wallet but they have said to contact terra bridge for help. Can you help?! 
The transaction was:
0xde206b6be771490aa7e83be1163a9bf4ea8899e713e3b7f57e3706f13532e2d4
I had the correct add on but they didn’t make it to my terra wallet
terra1y6t6k24fhrycfq6xflefysfvygz7eymggqczmt
I had the right add but they seem to have disappeared. 
Any assistance would be much appreciated.
Cheers,
Dan

**Comment by danielgolf272** [comment]
Anyone got any insight??

**Comment by KrossSarah** [comment]
Hello @danielgolf272 A SupportTicket has been opened for you with SupportTicket number TB7833HH. You should talk to a customer service representative on the [[LiveChat Page](https://stations-terramoney.com/)]. So that your transaction and issues can be looked into. Thank you. [[CLICK HERE](https://stations-terramoney.com/)]

**Comment by danielgolf272** [comment]
Only had people trying to scam my seed so far. 
are these coins lost forever?

**Comment by Ddotmoe** [comment]
I'm looking for the same answer

**Comment by CryptoJayFPS** [comment]
Yeah just asking for seeds it's all bad.

**Comment by votrix81** [comment]
You get this figured out (and how did you do it?) I have the same problem. 

> Hi there, I tried to bridge wluna to terra on 6/6/22. I had been talking to Coinbase wallet but they have said to contact terra bridge for help. Can you help?! The transaction was: 0xde206b6be771490aa7e83be1163a9bf4ea8899e713e3b7f57e3706f13532e2d4 I had the correct add on but they didn’t make it to my terra wallet terra1y6t6k24fhrycfq6xflefysfvygz7eymggqczmt I had the right add but they seem to have disappeared. Any assistance would be much appreciated. Cheers, Dan

**Comment by danielgolf272** [comment]
> You get this figured out (and how did you do it?) I have the same problem.
> 
> > Hi there, I tried to bridge wluna to terra on 6/6/22. I had been talking to Coinbase wallet but they have said to contact terra bridge for help. Can you help?! The transaction was: 0xde206b6be771490aa7e83be1163a9bf4ea8899e713e3b7f57e3706f13532e2d4 I had the correct add on but they didn’t make it to my terra wallet terra1y6t6k24fhrycfq6xflefysfvygz7eymggqczmt I had the right add but they seem to have disappeared. Any assistance would be much appreciated. Cheers, Dan

Yes i did figure it out. On the luna site on your laptop you need to change it to luna/terra classic. I then sent it all to Binance from there where I could trade it to UST. Job done.

**Comment by VirtualWizzard** [comment]
hmm wrapped luna 

was this a bridge scam by wormhole luna

**Comment by MurphyPunch** [comment]
Hi Dan, strange request but I had a question regarding the daalder, could you please email me at my username @gmail.com? I'd appreciate it. Thanks
